io/net/ethtool/ethtool_tests.sh.data/README.txt

```diff
@@ -0,0 +1,14 @@
+Tests the network driver and interface with 'ethtool' command.
+12 different options of ethtool command are tested for each
+of the interfaces specified in 'config' file
+
+Requirements:
+-------------
+For all specified interfaces, configuration file for that interface needs
+to be updated, so that 'ifup/ifdown <interface>' configures the interface.
+
+Input Needed (in 'config' file):
```

Comment:
...and this section. Clarify then how to use the multiplex file for the devices.